Exercise_A/index.html

```diff
@@ -20,26 +35,233 @@
+               <div class="image__overlay">
+                 <div class="image__title">FOUR THING</div>
+                 <p class="image__description">Bullshit</p>
+               </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+      
+    <aside class="bg-dark">
+      <div class="container text-center">
+        <div class="call-to-action">
+          <h2 class="cta-heading">Seriously, Its this fucking template</h2>
+          <button type="button" class="btn btn-light mt-4 rounded-pill animate__animated wow animate__jello">
+            That's Amazing
+          </button>
+        </div>
+      </div>
+    </aside>
     <!-- contact me container -->
+    <section id="callme">
+      <div class="col-lg-12 text-center mt-5">
+        <div class="row justify-content-center">
+          <div class="col-lg-8 col-lg-offset-2 text-center">
+            <h2>
+              My Contact Information
+            </h2>
+            <hr/>
+            <p>
+              Here's my phone number and email address, because hey that's part of the template and why would you not include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in from all the companies wanting my insane website design skills.
+
+Honestly this template does look really nice, though.
+            </p>
+     </div>
+     <div class="container">
+      <div class="row">
+        <div class="col">
+          <i class="fas fa-phone-alt fa-3x animate__animated wow animate__fadeInUp"></i>
+          <p class="pt-3">+1-608-317-1888</p>
+        </div>
+        <div class="col">
+          <i class="fas fa-envelope fa-3x animate__animated wow animate__fadeInUp"></i>
+          <p class="pt-3"><a href="mailto:tab@report.ly" class="email-links">holy@shit.com</a></p>
+        </div>
+      </div>
+    </section>
 
     <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
 
-    <!-- 
-      All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
-      You need the CDN for Smooth Scroll
-      Your custom JavaScript
-     -->
+    <script
+    src="https://code.jquery.com/jquery-3.5.1.js"
+    integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc="
+    crossorigin="anonymous"
+  ></script>
+
+  <script
+    src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+    crossorigin="anonymous"
+  ></script>
+    <script src="/Exercise_A/assets/js/wow.js"></script>
+    <script>
+    new WOW().init();
+    </script>
+    <script src="/Exercise_A/assets/js/script.js"></script>
   </body>
 </html>

```

Comment:
since you're using bootstrap version 4 CDN, it does not support data-bs-toggle & data-bs-target 
you can either use bootstrap 5 CDN or use data-toggle & data-target instead